app-5-rdf-serializer/src/test/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectOwlPropertiesTest.java

```diff
@@ -0,0 +1,449 @@
+                .setDeleted$1(false)
+                .build();
+        projectStatementWithLiteralTopic.pipeInput(kStLit, vStLit);
+
+        var kStLit2 = ProjectStatementKey.newBuilder().setProjectId(projectId).setStatementId(statementId).build();
+        var vStLit2 = ProjectStatementValue.newBuilder()
+                .setProjectId(projectId)
+                .setStatementId(statementId)
+                .setStatement(
+                        StatementEnrichedValue.newBuilder()
+                                .setSubjectId(subjectId)
+                                .setObjectId(objectId)
+                                .setPropertyId(propertyIdL2)
+                                .build())
+                .setDeleted$1(false)
+                .build();
+        projectStatementWithLiteralTopic.pipeInput(kStLit2, vStLit2);
+
+        var kStEnt = ProjectStatementKey.newBuilder().setProjectId(projectId).setStatementId(statementId).build();
+        var vStEnt = ProjectStatementValue.newBuilder()
+                .setProjectId(projectId)
+                .setStatementId(statementId)
+                .setStatement(
+                        StatementEnrichedValue.newBuilder()
+                                .setSubjectId(subjectId)
+                                .setObject(NodeValue.newBuilder()
+                                        .setClassId(0)
+                                        .build()
+                                )
+                                .setPropertyId(propertyIdE)
+                                .build())
+                .setDeleted$1(false)
+                .build();
+        projectStatementWithEntityTopic.pipeInput(kStEnt, vStEnt);
+
+        var kStEnt2 = ProjectStatementKey.newBuilder().setProjectId(projectId).setStatementId(statementId).build();
+        var vStEnt2 = ProjectStatementValue.newBuilder()
+                .setProjectId(projectId)
+                .setStatementId(statementId)
+                .setStatement(
+                        StatementEnrichedValue.newBuilder()
+                                .setSubjectId(subjectId)
+                                .setObject(NodeValue.newBuilder()
+                                        .setClassId(0)
+                                        .build()
+                                )
+                                .setPropertyId(propertyIdE2)
+                                .build())
+                .setDeleted$1(false)
+                .build();
+        projectStatementWithEntityTopic.pipeInput(kStEnt2, vStEnt2);
+
+        assertThat(outputTopic.isEmpty()).isFalse();
+        var outRecords = outputTopic.readKeyValuesToMap();
+
+
+        var expectedKey = ProjectRdfKey.newBuilder()
+                .setProjectId(projectId)
+                .setTurtle("<https://ontome.net/ontology/p123> a <http://www.w3.org/2002/07/owl#DatatypeProperty> .")
+                .build();
+        var expectedValue = ProjectRdfValue.newBuilder()
```

Comment:
the expectedValue1..13 are not used. Can be removed.

Reply:
Fixed